modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
I feel I should have named it `ortc/descriptionHandler.js` to avoid confusion :)
But it's ok, I'll do those changes. However, assuming a new naming for `parsed`, I don't fully understand why there shouldn't be a proper getter for it. The RTCSessionDescription shim will need to access to such a field so often.